src/Controller/ControllerFactory.php

```diff
@@ -321,7 +321,6 @@ protected function missingController(ServerRequest $request): MissingControllerE
             'plugin' => $request->getParam('plugin'),
             'prefix' => $request->getParam('prefix'),
             '_ext' => $request->getParam('_ext'),
-            'class' => $request->getParam('controller'), // Deprecated: Will be removed in 4.5. Use `controller` instead.
```

Comment:
actually, should this be done in 4.next and merged back to 5.x?

Reply:
I think removing this in 5.x is easier. Having breaking changes in 4.x doesn't feel great even if it is small stuff like this.